framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -169,7 +169,7 @@ MeshContinuum::MakeGridFaceHistogram(double master_tolerance, double slave_toler
     size_t running_face_count = 0;
     size_t running_face_size = face_size_histogram[0];
 
-    double running_average = (double)face_size_histogram[0];
+    auto running_average = (double)face_size_histogram[0];
```

Comment:
We should prefer `static_cast`.

Reply:
Clang-Tidy: Use auto when initializing with a cast to avoid duplicating the type name

is this incorrect?? @wdhawkins

Reply:
He is suggesting to do:
```
auto running_average = static_cast<double>(face_size_histogram[0]);
```
instead of the C-style cast...

Reply:
done